e2e/old-arch/panels-suite/geomap-map-controls.spec.ts

```diff
@@ -1,58 +1,71 @@
 import { e2e } from '../utils';
 const DASHBOARD_ID = 'P2jR04WVk';
+const TIMEOUT = 60000;
 
 describe('Geomap layer controls options', () => {
   beforeEach(() => {
+    cy.viewport(1920, 1080);
```

Comment:
i think this is already set as the default: https://github.com/grafana/grafana/blob/main/cypress.config.js#L13

Reply:
Great catch, I'll fix that

Reply:
Fixed 👍